subsystems/sidecar/index.js

```diff
@@ -537,15 +557,17 @@ class Sidecar extends ReadyResource {
 
     await sidecarClosed
 
-    for (const { dir, appling, cmdArgs, env } of restarts) {
+    const TERMINAL_RUNTIME = RUNTIME
```

Comment:
remove this, just use RUNTIME

Reply:
Tried this again. Same as before. It crashes when using RUNTIME when declaring RUNTIME in line 568 which I believe is due to hoisting.

Reply:
ok rename RUNTIME on 558 to TARGET_RUNTIME instead